examples/pxScene2d/src/pxResource.cpp

```diff
@@ -558,7 +558,7 @@ void pxResource::loadResource(rtObjectRef archive, bool reloading)
     mArchive = arc;
   }
   //rtLogDebug("rtImageResource::loadResource statusCode should be -1; is statusCode=%d\n",mLoadStatus.get<int32_t>("statusCode"));
-  if (mUrl.beginsWith("http:") || mUrl.beginsWith("https:"))
+  if (mUrl.beginsWith("http:") || mUrl.beginsWith("https:") || mUrl.beginsWith("file:"))
```

Comment:
Let's avoid using the rtFileDownloader for file:// urls.  Instead, let's load via loadResourceFromFile();.